SecurityPkg/SecurityPkg.dsc

```diff
@@ -81,12 +81,10 @@
   SourceDebugEnabledLib|SourceLevelDebugPkg/Library/SourceDebugEnabled/SourceDebugEnabledLib.inf ## MS_CHANGE_?
   Hash2CryptoLib|SecurityPkg/Library/BaseHash2CryptoLibNull/BaseHash2CryptoLibNull.inf   ## MU_CHANGE
   SafeIntLib |MdePkg/Library/BaseSafeIntLib/BaseSafeIntLib.inf ## MU_CHANGE - CodeQL change
-
-  ## MU_CHANGE TEMP
+  
 [LibraryClasses.X64, LibraryClasses.IA32]
-  ## NOTE: This is a temporary shim to resolve a build error while a more permanent solution is tested!
-  Tcg2PreUefiEventLogLib|SecurityPkg/Library/TempPreUefiEventLogLib/TempPreUefiEventLogLib.inf

```

Comment:
@kuqin12 & @apop5

Is there a reason to leave `SecurityPkg/Library/TempPreUefiEventLogLib/TempPreUefiEventLogLib.inf` in the tree?

I don't see it being used in any repos that I'm able to search.

Reply:
I do not think that TempPreUefiEventLogLib is worth keeping in the tree since the functionality can be replaced with the Null lib.

If you don't see it used in any of the repos, then I think it might be safe to remove it.

Reply:
I think so too. I vaguely remember one of our proprietary hardware platforms has an override of this instance... But we should not care about that specific case.

Reply:
Should I remove TempPreUefiEventLogLib in this PR and push the changes?

Reply:
I think it is safe to leave it out of this PR.

We can create a separate PR that removes it for better visibility after.

Reply:
Sure